src/components/interactive-builder/modals/question/question-form/rendering-types/inputs/date/date.component.tsx

```diff
@@ -25,7 +25,7 @@ const Date: React.FC = () => {
   );
 
   return (
-    <RadioButtonGroup name="datePickerType" legendText={t('datePickerType', 'The type of date picker to show ')}>
+    <RadioButtonGroup name="datePickerType" defaultSelected="both" legendText={t('datePickerType', 'The type of date picker to show ')}>
```

Comment:
>  using 'defaultSelected' property of RadioButtonGroup would be a better way to set the default value for Date picker type, made the required changes as well.

You can use `defaultSelected` but that would mean that the user gets the impression that when they save the defaultSelected value would be persisted, but that isn't the case here. To fix that, referring back to my first suggestion, you'd have to set the `datePickerFormat` in the `formField`